virt/virt-web-console-overview.adoc

```diff
@@ -623,7 +628,214 @@ You can create, edit, and clone VirtualMachine templates on the Templates page.
 |Search field |Search for templates by name or by label.
 |*Templates* table |List of templates.
 
-Click the Options menu {kebab} beside a template to select *Edit*, *Clone*, *Edit boot source*, *Edit boot source reference*, *Edit labels*, *Edit annotations* or *Delete*.
+Click the Options menu {kebab} beside a template to select *Edit*, *Clone*, *Edit boot source*, *Edit boot source reference*, *Edit labels*, *Edit annotations*, or *Delete*.
+|====
+=====
+
+[id="ui-template-details-page_{context}"]
+=== Template details page
+
+You can view template settings and edit custom templates on the Template details page.
+
+.Template details page
+[%collapsible]
+=====
+[cols="1,3", options="header"]
+|====
+|Element |Description
+
+|*Actions* menu
+|Click the *Actions* menu to select *Edit*, *Clone*, *Edit boot source*, *Edit boot source reference*, *Edit labels*, *Edit annotations*, or *Delete*.
+
+|xref:../virt/virt-web-console-overview.adoc#ui-template-details-details_virt-web-console-overview[*Details* tab]
+|Template settings and configurations.
+
+|xref:../virt/virt-web-console-overview.adoc#ui-template-details-yaml_virt-web-console-overview[*YAML* tab]
+|YAML configuration file.
+
+|xref:../virt/virt-web-console-overview.adoc#ui-template-details-scheduling_virt-web-console-overview[*Scheduling* tab]
+|Scheduling configurations.
+
+|xref:../virt/virt-web-console-overview.adoc#ui-template-details-network-interfaces_virt-web-console-overview[*Network interfaces* tab]
+|Network interface management.
+
+|xref:../virt/virt-web-console-overview.adoc#ui-template-details-disks_virt-web-console-overview[*Disks* tab]
+|Disk management.
+
+|xref:../virt/virt-web-console-overview.adoc#ui-template-details-scripts_virt-web-console-overview[*Scripts* tab]
+|Cloud-init, SSH key, and Sysprep management.
+
+|xref:../virt/virt-web-console-overview.adoc#ui-template-details-parameters_virt-web-console-overview[*Parameters* tab]
+|VirtualMachine name, data source name and namespace, and cloud user password management.
+|====
+=====
+
+[id="ui-template-details-details_{context}"]
+==== Details tab
+
+You can configure a custom template on the *Details* tab.
+
+.Details tab
+[%collapsible]
+=====
+[cols="1,3a", options="header"]
+|====
+|Element |Description
+|*YAML* switch |Select *ON* to view your live changes in the YAML configuration file.
+|Name |Template name.
+|Namespace |Template namespace.
+|Labels |Click the edit icon to edit the labels.
```

Comment:
Icons that are not 'named' or do not have tooltips are not marked up.

Reply:
I guess in this instance whether or not the edit icon opens up a menu is what we should consider. Since `labels` implies multiple actions could be taken, I thought the rule about bolding icons that open menus should be applied. 

FWIW, I saw the `click the print icon` and felt this was a different case. https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#slider